src/locales/en-US/commands.json

```diff
@@ -672,6 +672,15 @@
     "commandUsage": "<text>",
     "noText": "You have to give me some text to asciify!"
   },
+  "itunes": {
+    "commandDescription": "Provides the content within the iTunes Store and Apple Books Store.",
+    "commandUsage": "<media> <term> <country code>",
+    "title": "Showing Results for {{title}}.",
+    "noMedia": "You will have to give me a valid type of media",
+    "availableMedia": "All the available media's are",
+    "noText": "You have to give me media and search terms!",
```

Comment:
Either use this in the missingError option in the second argument or remove this